### PR TITLE
Fix string labels in performance report / simple model comparison 

### PR DESCRIPTION
**Pull request by JKL98ISR**

### CODE REVIEW

File: deepchecks/checks/performance/performance_report.py [hunk 1]
```diff
@@ -97,14 +97,14 @@ def _performance_report(self, train_dataset: Dataset, test_dataset: Dataset, mod
         validate_model(test_dataset, model)
 
         task_type = task_type_check(model, train_dataset)
-        clasess = test_dataset.classes
+        clasess = train_dataset.classes
```
Comment (matanper) [inline deepchecks/checks/performance/performance_report.py:100]: why did you change to train?

Comment (JKL98ISR) [reply deepchecks/checks/performance/performance_report.py:100]: test can contain classes not in train

Comment (JKL98ISR) [reply deepchecks/checks/performance/performance_report.py:100]: missing labels from train should do an error either way no?

Comment (JKL98ISR) [reply deepchecks/checks/performance/performance_report.py:100]: not an error just bad score*

**Review by matanper — APPROVED** [review]
